bip-0360.mediawiki

```diff
@@ -641,6 +641,38 @@ proposed solution] in an Ethereum quantum emergency is quite different from the
 a hard fork of the chain, reverting all blocks after a sufficient amount of theft, and using STARKs based on BIP-32
 seeds to act as the authoritative secret when signing. These measures are deemed far too heavy-handed for Bitcoin.
 
+P2QRH and MAST (Merkelized Abstract Syntax Tree) [https://github.com/bitcoin/bips/blob/master/bip-0114.mediawiki BIP-114],
+and related BIPs [https://github.com/bitcoin/bips/blob/master/bip-0116.mediawiki BIP-116], [https://github.com/bitcoin/bips/blob/master/bip-0117.mediawiki BIP-117],
+share the idea of committing to a Merkle tree of scripts. While MAST was never activated, taproot
+[https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki BIP-341] incorporated this idea of a Merkle tree of
+scripts into its design. For the purposes of compatibility, simplicity and code reuse P2QRH inherits this capability from taproot
+rather than implementing MAST directly.
```

Comment:
It also be noted here, for clarity (even if it's redundant, it might be the only thing someone reads and it hopefully will reduce confusion), that the major difference between P2TR and P2QRH is that key path spends are removed and as a result of removing the key, we no longer need to compute it or tweak it, so tap tweaks are replaced by the quantum root, which is just a commitment to the Merkle tree of TapScripts.